src/constants/propTypes.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import { PATIENT_CODE_REGEX } from "./constants";
+
+const CodeProp = function (props, propName, componentName) {
+  if (
+    !PATIENT_CODE_REGEX.test(
+      // eslint-disable-next-line react/destructuring-assignment
+      props[propName]
+    )
+  ) {
+    return new Error(
+      `Invalid Code: \`${propName}\` supplied to` +
+        ` \`${componentName}\`. Validation failed.`
+    );
+  }
+  return null;
+};
+
+const Patient = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  surname: PropTypes.string.isRequired,
+  sex: PropTypes.string.isRequired,
+  phone: PropTypes.string.isRequired,
+  weight: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+  bmi: PropTypes.number.isRequired,
+  diagnosis: PropTypes.arrayOf(PropTypes.string),
+  medicines: PropTypes.arrayOf(PropTypes.string),
+  hospitalization_times: PropTypes.number,
+  pesel: PropTypes.string.isRequired,
+  date_of_birth: PropTypes.string.isRequired,
```

Comment:
Czy te wszystkie pola oznaczone jako isRequired są rzeczywiście wymagane?

Reply:
Jeszcze to przeglądne, ale trudno mi to do końca stwierdzić. To już bardziej pytanie do Dawida. Na razie wyszedłem z założenia, że lepiej jak będziemy czegoś wymagać i najwyżej później damy to jako opcjonalne.